test/commands/extended/get_bundles_test.py

```diff
@@ -23,18 +23,24 @@ def setUp(self):
         super(GetBundlesRequestFilterTestCase, self).setUp()
 
         # noinspection SpellCheckingInspection
-        self.transaction = (
-            'TESTVALUE9DONTUSEINPRODUCTION99999KPZOTR'
-            'VDB9GZDJGZSSDCBIX9QOK9PAV9RMDBGDXLDTIZTWQ'
-        )
+        self.transactions = [
+            (
+                'TESTVALUE9DONTUSEINPRODUCTION99999KPZOTR'
+                'VDB9GZDJGZSSDCBIX9QOK9PAV9RMDBGDXLDTIZTWQ'
+            ),
+            (
+                'TESTVALUE9DONTUSEINPRODUCTION99999KPZOTR'
+                'VDB9GZDJGZSSDCBIX9QOK9PAV9RMDBGDXLDTIZTWQ'
+            ),
```

Comment:
❔ Should these values be different, to more accurately simulate real-world usage?

Reply:
agreed